cli/config_file.rs

```diff
@@ -73,7 +73,6 @@ pub const IGNORED_COMPILER_OPTIONS: &[&str] = &[
   "esModuleInterop",
   "emitDeclarationOnly",
   "importHelpers",
-  "inlineSourceMap",
```

Comment:
Hmmm... there will still should be reported as ignored _unless_ your doing `deno bundle`.  Maybe we need another set, which is only analyzed when your are bundling.